### ISSUE TITLE
on every start/restart warning: Operand of null-aware operation

### ISSUE THREAD

**Issue by MohamedEL-Torky** [Baseflow/flutter_cache_manager#299]
## 🐛 Bug Report

I'm using Flutter cached network image which depends on this package.
on every hot restart or cold start, I'm getting this error message from the internals of this package.

```
../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_cache_manager-3.0.0-nullsafety.1/lib/src/storage/cache_info_repositories/cache_object_provider.dart:208:29: Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
    final oldDbPath = join((await getDatabasesPath())!, '$databaseName.db');

```

**Version:** 3.0.0-nullsafety.1

**Platform:** 
- [✓] :iphone: iOS
- [✓] :robot: Android

**Comment by itmediacreations** [comment]
Just remove `!` operand from file flutter_cache_manager-3.0.0-nullsafety.1/lib/src/storage/cache_info_repositories/cache_object_provider.dart in line 208:29 till official update come, it occurs after you upgrade sqflite to 2.0.0+3

**Comment by ShahoG** [comment]
Can confirm it happens on Android aswell

**Comment by MohamedEL-Torky** [comment]
@ShahoG Thanks, I've edited the issue report.

**Comment by zhukeev** [comment]
The same 
/.pub-cache/hosted/pub.dartlang.org/flutter_cache_manager-3.0.0-nullsafety.1/lib/src/storage/cache_info_repositories/cache_object_provider.dart:208:29: Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
    final oldDbPath = join((await getDatabasesPath())!, '$databaseName.db');
                            ^

**Comment by knnkrt** [comment]
Same issue. How to solve this.

**Comment by renefloor** [comment]
Fixed in 3.0.0-nullsafety.2
